Earthfile

```diff
@@ -145,7 +145,7 @@ frontend-test:
   COPY conf/kiterc_template conf/
   COPY test/localhost.self-signed.cert* test/
   ENV CI true
-  RUN cd web && ../tools/with_lk.sh yarn playwright test || touch failed
+  RUN cd web && ../tools/with_lk.sh > /dev/null 2>&1 yarn playwright test || touch failed
```

Comment:
Come on, I even said to add the redirect _in_ `with_lk.sh`. I also said to test it locally. But even if you didn't, you can look at the GitHub Actions output and see this is wrong.

I think you're very confused about something, so let me break down what's happening:

```python
RUN
  cd web
&&
  ../tools/with_lk.sh yarn playwright test
||
  touch failed
```

Three commands with logical operators between them. Each command is a space-separated list where the first item is an executable (or shell built-in) and the rest are its arguments. Specifically in the second command we run `with_lk.sh` with the `yarn`, `playwright`, `test` arguments.

What happens if you stick `> /dev/null 2>&1` in the middle of the argument list? I've never seen anything like this, but it looks like it's the same as sticking it at the end:

```bash
$ echo x > y z
$ cat y
x z
```

Wild!

Perhaps you don't know what `with_lk.sh` does? It has a comment explaining it and it's not a lot of code. But maybe we never discussed it. It starts LynxKite in the background, runs the command it got as an argument, then shuts down LynxKite.

What your change does is mute this whole step. So LynxKite is successfully muted, but you're also muting Playwright!

I'm certainly dumber and slower when I have a cold, so all dumbness and slowness for both of us can be excused this week. You use Linux as your primary system, we built the Earthly setup and the Playwright setup together, and you reviewed major changes in `with_lk.sh`, so I think you understand all this. But let me know if we need to go over some things.